### PR TITLE
Added YouTube video ID to talks and embedded videos for past talks

### PR DESCRIPTION
**Pull request by opiation**
The pull request introduces embedded YouTube videos for past talks as proposed in #277.

This feature does the following:

- Adds a `youtube_id` string column to the **Talk** model
- Adds the `youtube_id` field to the talk dashboard
- Adds the "YouTube ID" label to the I18n config/locales/en.yml
- Adds an embedded `<iframe>` video if the talk is both in the past **and** has a non-nil `youtube_id`

### CODE REVIEW

**Comment by opiation** [comment]
Should I presume from the above coverage decrease alert that that PR will not be accepted on those grounds?

**Comment by sophiedeziel** [comment]
Yep. I can do the review in parallel, but coverage requirement has to be met.

**Comment by opiation** [comment]
No problem.  I'll add some coverage tonight.

File: app/views/talks/_talk.html.erb [hunk 1]
```diff
@@ -9,6 +9,11 @@
 
       <h4><%= render_markdown_as_html "#{talk.title.titleize}, by #{talk.member.name}" %></h4>
 
+      <% if talk.event&.starts_at&.past? && talk.youtube_id.present? %>
+        <iframe src="https://www.youtube.com/embed/<%= talk.youtube_id %>?rel=0&autoplay=0" frameborder="0" allowfullscreen width="640" height="360"></iframe>
```
Comment (sophiedeziel) [inline app/views/talks/_talk.html.erb]: This pull-request is pretty simple and very good. The only suggestion I would make is create a helper for the youtube tag

**Comment by sophiedeziel** [comment]
I'm merging, even with my suggestion. I understand that you might not have the time to make that change and this is adding a lot of value to the project as is. Thank you for opening that pull-request!

**Pull request merged**